src/main/java/com/fasterxml/jackson/databind/deser/std/FactoryBasedEnumDeserializer.java

```diff
@@ -202,6 +224,10 @@ public Object deserialize(JsonParser p, DeserializationContext ctxt) throws IOEx
                 if (ctxt.isEnabled(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_AS_NULL)) {
                     return null;
                 }
+
+                if (ctxt.isEnabled(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE)) {
```

Comment:
I'll move this before `as null`... plus, should only return non-null `_defaultValue`, I think.